inspector/scripts/requestAndRead.ts

```diff
@@ -0,0 +1,105 @@
+import { deployments, ethers, getNamedAccounts } from "hardhat";
+import {
+  getKeyHash,
+  estimateVRFServiceFee,
+  estimateRRServiceFee,
+  getAggregatorPairs,
+  priceFeeds,
+  findPairsWithSameRoundId,
+} from "./utils";
+
+async function main() {
+  const {
+    sResponse: rrResponseBefore,
+    sRandomWord: vrfResponseBefore,
+    priceFeedResults: priceFeedsBefore,
+  } = await read();
+
+  await request();
+
+  // wait 30 seconds for fulfillment and submission
```

Comment:
I think we should wait much less than 30 seconds. If the service does not respond fast, there is some issue. Let's set it up to 5 seconds and aim to keep it under 3 mid-term.